### PR TITLE
feat(links): [MAG-3268] Link lists for header, plus Logo component

### PR DESCRIPTION
**Pull request by matthill8286**
# <https://filtered-ewb.atlassian.net/browse/MAG-3268 / Link Lists>

## TL;DR

-   Link lists, story, tests
-   Logo, story, tests

Screenshot 2020-12-14 at 15 09 06

Screenshot 2020-12-14 at 14 54 46

## Details

Checklist / Reminders:

-   [x] Documentation / Story is created / adapted
-   [x] Unit tests
-   [x] CSS changes are BEM-compliant

### CODE REVIEW

File: src/components/Atoms/Logo/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  color?: ThemeColors
+}
+
+const StyledWrapper = styled.div<StyledWrapperProps>`
+  position: relative;
+  max-width: 200px;
+  fill: ${({ color, theme }) => (color && theme.color[color]) || theme.color.primary};
+`
+
+// const StyledFilteredLogo = styled(SvgFilteredLogo)`
```
Comment (tricaalexandru) [inline src/components/Atoms/Logo/Logo.tsx:23]: @matthill8286 commented out code.

File: src/components/Atoms/Logo/Logo.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+//   display: block;
+//   height: 80px;
+//   width: 100%;
+// `
+
+export const Logo: FunctionComponent<LogoProps> = ({ white, className, uni }) => {
+  return (
+    <StyledWrapper color={white ? 'white' : undefined} className={className}>
+      <Picture src="public/images/logos/filtered.png" alt="thumbnail" width="180" />
```
Comment (tricaalexandru) [inline src/components/Atoms/Logo/Logo.tsx:32]: @matthill8286 is this left here by mistake, for a test or is it a fallback?

**Review by tricaalexandru — APPROVED** [review]

File: src/components/Atoms/Logo/Logo.tsx (hunk 1 above)
Comment (matthill8286) [reply src/components/Atoms/Logo/Logo.tsx:23]: @tricaalexandru need to find a way of converting images to svgs then we can use this part

File: src/components/Atoms/Logo/Logo.tsx (hunk 2 above)
Comment (matthill8286) [reply src/components/Atoms/Logo/Logo.tsx:32]: @tricaalexandru This Logo component is really only a test, ideally we should be serving svgs, so it's only a mocked component, it's not exported from the component library 🔢

**Pull request merged**